app/screens/sso/sso.js

```diff
@@ -20,7 +20,20 @@ import StatusBar from 'app/components/status_bar';
 import PushNotifications from 'app/push_notifications';
 import {changeOpacity, makeStyleSheetFromTheme} from 'app/utils/theme';
 
-const jsCode = 'window.postMessage(document.body.innerText)';
+const postMessageJS = 'window.postMessage(document.body.innerText);';
```

Comment:
This code changed for release 1.3 perhaps we should wait until that is merged back into master?